projects/plugins/boost/app/assets/src/js/utils/generate-critical-css.ts

```diff
@@ -154,10 +166,14 @@ async function generateForKeys(
 	const majorSteps = Object.keys( providerKeys ).length + 1;
 	let majorStep = 0;
 
+	// eslint-disable-next-line
```

Comment:
Please always specify which linting rule is being disabled. e.g.:

```suggestion
	// eslint-disable-next-line @wordpress/no-unused-vars-before-return
```

It makes it clearer why the exception exists when others are reading the code later.

🤔  TBH I question the value of this linting rule, but that can/should be discussed outside the context of this PR.